lib/Cake/Test/Case/Routing/DispatcherTest.php

```diff
@@ -1605,6 +1605,12 @@ public static function cacheActionProvider() {
  * @return void
  */
 	public function testFullPageCachingDispatch($url) {
+		// PHP 8.0+ has a bug in which the expiration date is not cached
```

Comment:
Could you elaborate on that? I've never heard about such problem.

Reply:
This test fails with the following error at the last `unlink($filename);`.

```
unlink(/var/www/html/app/tmp/cache/views/var_www_html_vendors_phpunit_phpunit_index.php): No such file or directory
 /var/www/html/lib/Cake/Test/Case/Routing/DispatcherTest.php:1645
```

If a relative time such as `'+2 hours'` is specified, the following conditional expression is false and is no longer cached due to a change in the behavior of the comparison operator.

https://github.com/kamilwylegala/cakephp2-php8/blob/9e0fd38f37adc88e9d44debc98f485490658291c/lib/Cake/View/Helper/CacheHelper.php#L152